### ISSUE TITLE
Mistakable behaviour when using the mouse back button

### ISSUE THREAD

**Issue by l-e-e-o** [bitwarden/desktop#314]
I just discovered that after logging in I can use the back button of my mouse to return to the password prompt.  The problem with this is that the vault looks locked when in fact I can return to the open vault by using the forward button of my mouse.  

I've tested this in Version 1.16.4 on Linux and version 1.41.0 of the Chrome plugin.  My mouse sends a button 8 event for back and a button 9 event for forward.  It does however not work with the XF86Back and XF86Forward events.

**Comment by dario-piotrowicz** [comment]
Hi, I have tested this on the current desktop version (1.26.5) on Linux (Ubuntu 20.04.2 LTS) and there does not seem to be such issue.

I checked and my mouse also sends button 8 and 9 events for back and forward respectively.

So I believe that the bug is no longer present and would suggest double checking this issue and close it if I'm actually right.

It is however present in the browser extension.
I will raise an issue in the repo (https://github.com/bitwarden/browser) for this.
